astropy/utils/tests/test_data.py

```diff
@@ -1260,6 +1262,8 @@ def test_download_cache_update_doesnt_damage_cache(temp_cache, valid_urls):
     download_file(u, cache="update")
 
 
+@pytest.mark.filterwarnings(
+    'ignore:Remote data cache could not be accessed')
```

Comment:
It's weird that this is needed; I thought all such warnings would be caught and suppressed by the `pytest.warns` context managers. I guess I missed some?